src/main/java/com/achdev/onlinebookstoreapp/controller/BookController.java

```diff
@@ -25,8 +27,9 @@ public class BookController {
     private final BookService bookService;
 
     @GetMapping
-    public List<BookDto> getAll() {
-        return bookService.findAll();
+    public PageResponse<BookDto> getAll(Pageable pageable) {
```

Comment:
Add Swagger annotations for each endpoint

Reply:
Ok